.github/workflows/publish-go-tester-task.yml

```diff
@@ -159,3 +159,35 @@ jobs:
         with:
           path: ./*checksums.txt
           name: checksums
+
+  proto-files:
+    # Run this after checksums or the uploaded proto files zip
+    # will make the checksum job always fail
+    needs:
+      - checksums
+      - package-name-prefix
```

Comment:
Why `checksums` is failing? shouldn't just add the checksum of the protoc archive to the checksums file together with the other archives?

Reply:
I didn't think about that, it would make sense since it's a file part of the release. :thinking: 
I'd have to edit a bit the checksum process I think, I'll take a look at it.